README.md

```diff
@@ -10,4 +10,61 @@ I will not be guarenting anything. I'll try to target both bluesky & mastodon, b
 
 ### This is not (and probably won't be for a while) a 100% accurate recreation of the twitter API
 
-most of this readme todo
\ No newline at end of file
+most of this readme todo
+
+## Compatibility
```

Comment:
I feel like if we are to put a list of as many devices as possible, we should do it in a separate file (for things like Tizen TV) and add a summary of the most common devices in the readme

Reply:
Also categories would be nice

Reply:
Yes

Reply:
You can edit yourself